utest/api/test_languages.py

```diff
@@ -102,6 +119,40 @@ def test_duplicates_are_not_added(self):
         langs.add_language('th')
         assert_equal(list(langs), [Fi(), En(), PtBr(), Th()])
 
+    def test_get_available_languages(self):
+        languages = Languages.get_available_languages()
+        self.assertIn(("en", En), languages.items())
+        self.assertIn(("english", En), languages.items())
+        self.assertIn(("fi", Fi), languages.items())
+        self.assertIn(("finnish", Fi), languages.items())
+
+    def test_import_language_module(self):
+        data = join(abspath(dirname(__file__)), 'elvish_languages.py')
+        languages = Languages.import_languages_module(data)
+
+        self.assertIn("elvsin", languages)
+        self.assertIn("elvishsindarin", languages)
+        self.assertIn("elvque", languages)
+        self.assertIn("elvishquenya", languages)
+
+        avail_languages = Languages.get_available_languages()
+
+        self.assertIn("elvsin", avail_languages)
+        self.assertIn("elvishsindarin", avail_languages)
+        self.assertIn("elvque", avail_languages)
+        self.assertIn("elvishquenya", avail_languages)
+
+    def test_init_with_language_module(self):
+        data = join(abspath(dirname(__file__)), 'orcish_languages.py')
+
+        languages = [(v.name, v.code) for v in Languages(data)]
+
+        self.assertIn(("OrcishLoud", "or-CLOU"), languages)
+        self.assertIn(("Orcish Quiet", "or-CQUI"), languages)
+        self.assertIn(("English", "en"), languages)
+
+
+
```

Comment:
:+1: for comprehensive unit tests. Unnecessary empty lines should be removed, though.